products/apache-airflow.md

```diff
@@ -10,42 +10,56 @@ versionCommand: airflow version
 releasePolicyLink: https://github.com/apache/airflow#version-life-cycle
 changelogTemplate: "https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-{{'__LATEST__'|replace:'.','-'}}-__LATEST_RELEASE_DATE__"
 releaseDateColumn: true
-
-auto:
-  methods:
-  -   pypi: apache-airflow
+eolColumn: Limited Support
```

Comment:
Make support visible (diffrent than eol for cycle 2), use same naming as in https://github.com/apache/airflow#version-life-cycle:
```suggestion
eolColumn: EOL/Terminated
activeSupportColumn: Limited Support
```

Reply:
Oh, looks like I did all the changes but forgot to display the `activeSupport` column ! I fixed it.

`Limited Support` is the phase between the `Limited Support` date and the `EOL/Terminated` date. So the `activeSupport` column cannot be named  `Limited Support` IMO. That's also why I name the `activeSupport` phase just `Active support`.